packages/framework/aqueduct/src/data-objects/pureDataObject.ts

```diff
@@ -29,7 +28,7 @@ import { DataObjectTypes, IDataObjectProps } from "./types";
  */
 export abstract class PureDataObject<I extends DataObjectTypes = DataObjectTypes>
 	extends TypedEventEmitter<I["Events"] & IEvent>
-	implements IFluidLoadable, IFluidRouter, IProvideFluidHandle
+	implements IFluidLoadable, IProvideFluidHandle
```

Comment:
I get how this is not technically breaking because the interface members are still there... but I feel it would be clearer to not remove it just yet and do it in a single step together with the removal of the functional code. Keep this change limited to purely adding deprecation notices and linter overrides.

Reply:
+1 to this - my pref is to remove the members along with the implementation declaration.